setup.py

```diff
@@ -9,9 +9,9 @@
 install_requires = [
     'click==6.6',
     'botocore>=1.5.40,<2.0.0',
-    'virtualenv>=15.0.0,<16.0.0',
     'typing==3.5.3.0',
     'six>=1.10.0,<2.0.0',
+    'pip>=9'
```

Comment:
Once or twice a year seems reasonable, I'm just concerned that it's still a possibility within semver to break a common API (even if it's rare), and people on older versions of chalice won't be guarded against this.